translations/src/config.js

```diff
@@ -33,8 +33,8 @@ class Config {
     tr: { rtl: false },
     pl: { rtl: false },
     ti: { rtl: false },
-    ku: {
-      rtl: false,
+    ckb: {
+      rtl: true,
```

Comment:
Sorani, which uses the persian alphabet is in fact a RTL language. However the  translations we have currently don't have the translations in persian alphabet. So do the translations also have to change then?

Reply:
Our current translations are probably `kmr` then. Could you verify that and rename them to `kmr`? `ckb` translations have to be created with our next translation process then.

Reply:
Ah thank you very much. I was really confused by those alphabets.